backend/src/middleware/permissionsMiddleware.js

```diff
@@ -68,7 +68,11 @@ const isAuthor = rule({
 const isDeletingOwnAccount = rule({
   cache: 'no_cache',
 })(async (parent, args, context, info) => {
-  return context.user.id === args.id
+  if (isAdmin === true) {
+    return true
```

Comment:
This is a little bit odd for me and fits even not to the name of the constant `isDeletingOwnAccount`.